### ISSUE TITLE
Command PhaseScriptExecution failed with a nonzero exit code in SalesforceSDKCommon

### ISSUE THREAD

**Issue by timcolla** [forcedotcom/SalesforceMobileSDK-iOS#2829]
TL;DR Quick fix: Don't have spaces in your root project directory

1. Version of Mobile SDK Used: 
7.0

2. Issue found in Native App or Hybrid App:
Native

3. OS Version:
N/A

4. Device:
N/A

5. Steps to reproduce:
Have the root directory of the project contain spaces

6. Actual behavior:
Project doesn't build

7. Expected Behavior:
Project builds

8. Error Log:
/bin/sh -c /Users/timcolla/Library/Developer/Xcode/DerivedData/project_name_with_spaces-empotddjncgkatftxlkczefcffdh/Build/Intermediates.noindex/SalesforceSDKCommon.build/Debug-iphonesimulator/SalesforceSDKCommon.build/Script-B7D55E412181696500827C89.sh

usage: dirname path
find: project: No such file or directory
find: name: No such file or directory
find: with: No such file or directory
find: spaces/SalesforceMobileSDK-iOS/libs/SalesforceSDKCommon: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code

Problem occurs in "shared/common/build/tools/update_podspec_headers.sh" on line 76
```
repoHeaderFile=`find ${projectDir} -name $headerFile`
```

When ${projectDir} contains spaces this error happens. My bash skills aren't good enough to fix this, but I believe ${projectDir} should be in double quotes to allow spaces.

I hope this is enough information for it to be fixed :)

Thanks,
Tim

**Comment by wmathurin** [comment]
How are you running into that issue? How are you consuming the Mobile SDK?

**Comment by timcolla** [comment]
Hi,

I followed the "Creating an iOS Swift Project Manually" guide.
https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/ios_new_native_project_manual.htm

Is that what you mean?

Thanks,
Tim

**Comment by wmathurin** [comment]
I don't think spaces in the project name is a good idea (make doesn't like it etc - for more read https://stackoverflow.com/questions/28914120/spaces-in-xcode-project-names-whats-the-best-practice).

I should also point out there is a less manual way of building apps for Mobile SDK (see https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/ios_new_force_project.htm). `forceios` won't let you create project names with spaces (and does other validations as well).

**Comment by timcolla** [comment]
That's fair enough. Thanks for the reply!
